translations/handleiOSNotesTranslations.sh

```diff
@@ -13,41 +13,17 @@ git clone git@github.com:nextcloud/notes-ios /app
 git checkout -b develop origin/develop
 
 # remove all translations (they are added afterwards anyways but allows to remove languages via transifex)
-rm -r Source/*.lproj/Localizable.strings
-rm -r Source/*.lproj/Main_iPhone.strings
-rm -r Source/*.lproj/Categories.strings
-rm -r Source/Settings.bundle/*.lproj/Root.strings
-rm -r Source/Screens/Settings/*.lproj/Settings.strings
-
-git checkout -- Source/en.lproj/Localizable.strings
-git checkout -- Source/en.lproj/Main_iPhone.strings
-git checkout -- Source/en.lproj/Categories.strings
-git checkout -- Source/Settings.bundle/en.lproj/Root.strings
-git checkout -- Source/Screens/Settings/en.lproj/Settings.strings
-
+rm -rf Source/*.lproj
+git checkout -- Source/en.lproj
 
 # push sources
 tx push -s
 
 # pull translations
 tx pull -f -a --minimum-perc=50
 
-
-# use de_DE instead of de
-rm -rf Source/de.lproj/Localizable.strings
-rm -rf Source/de.lproj/Main_iPhone.strings
-rm -rf Source/de.lproj/Categories.strings
-rm -rf Source/Settings.bundle/de.lproj/Root.strings
-rm -rf Source/Screens/Settings/de.lproj/Settings.strings
-
-mv Source/de_DE.lproj/Localizable.strings Source/de.lproj/Localizable.strings
-mv Source/de_DE.lproj/Main_iPhone.strings Source/de.lproj/Main_iPhone.strings
-mv Source/de_DE.lproj/Categories.strings Source/de.lproj/Categories.strings
-mv Source/Settings.bundle/de_DE.lproj/Root.strings Source/Settings.bundle/de.lproj/Root.strings
-mv Source/Screens/Settings/de_DE.lproj/Settings.strings Source/Screens/Settings/de.lproj/Settings.strings
-
 # create git commit and push it
 git add .
-git commit -am "[tx-robot] updated from transifex" || true
+git commit -am "Updated localizations from Transifex" -s || true
```

Comment:
Let's stay aligned with conventional commits:
```suggestion
git commit -am "fix(l10n): Updated translations from Transifex" -s || true
```

Reply:
Which convention? 😅 The three iOS projects I have seen all had their own thing going on. Talk for iOS uses a different capitalization:

> Fix(l10n): Update translations from Transifex

It is the same as the Nextcloud server repository uses, so I will update this to that.

Being new, "Fix" appeared semantically inappropriate to me because it implies something was broken which is not always the case with updated localization resources.

Reply:
See https://github.com/nextcloud/docker-ci/pull/732/commits/b17fd7ee3b873e6a64cc700d8b128c981671051a